cob_bringup/components/cob4_base.launch

```diff
@@ -3,6 +3,7 @@
 	<arg name="robot"/>
```

Comment:
I gave @destogl @andreeatulbure a heads-up https://github.com/ipa320/cob_robots/pull/609#issuecomment-281258827 about this...we will merge this week - with or without #609